### PR TITLE
Fixed Browser Card dark mode in index2.html and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by gurvirlochab**
1. Fixed Browser card in index2.html that wasn't reflecting dark mode
2. Fixed some typos in CONTRIBUTING.md

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -21,17 +21,17 @@ Contributions are always **welcome and recommended**! Here is how for beginner's
...
 ## Online one-click setup for contributing
 
-You can use Gitpod(an online IDE which is free for Open Source) for working on issues or making Prs. With a single click it will launch a workspace and automatically:
+You can use Gitpod(an online IDE which is free for Open Source) for working on issues or making PRs. With a single click it will launch a workspace and automatically:
 
 - clone the `AdminLTE` repo.
 - install the dependencies.
 - run `npm run dev` to start the server.
 
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/)
```
Comment (danny007in) [inline .github/CONTRIBUTING.md:37]: revert this line

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -21,17 +21,17 @@ Contributions are always **welcome and recommended**! Here is how for beginner's
     * Delete ./package-lock.json file
     * `npm install` (install npm deps)
     * `npm run dev` (developer mode, autocompile with browsersync support for live demo)
-    * Make your changes only in ./buid Folder OR package.json OR ./dist/js/demo.js OR in any html files which nessary to contribute
-    * Do not changes in ./dist/css/ AND ./dist/js/ Because its compiled files
-    * `npm run production` (compile css/js files and test every pages are perfectly working fine, before creating pull request)
+    * Make your changes are only in ./buid Folder OR package.json OR ./dist/js/demo.js OR in any html files which are necessary to contribute to
```
Comment (danny007in) [inline .github/CONTRIBUTING.md:24]: - Rename `./buid` to `./build`
- add `package-lock.json OR ./dist/js/pages/ OR ./docs` to this line
- change `are necessary to contribute to` to `are necessary for contribution`

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -21,17 +21,17 @@ Contributions are always **welcome and recommended**! Here is how for beginner's
...
+    * Do not make changes in ./dist/css/ AND ./dist/js/ Because it contains compiled files
```
Comment (danny007in) [inline .github/CONTRIBUTING.md:25]: - add `./plugins` in this line
- add `also do not include in PR (Pull Request)` in this line

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -21,17 +21,17 @@ Contributions are always **welcome and recommended**! Here is how for beginner's
...
+    * `npm run production` (compile css/js files and test all pages are perfectly working fine, before creating a pull request)
 4. Create a pull request
 
 ## Online one-click setup for contributing
 
-You can use Gitpod(an online IDE which is free for Open Source) for working on issues or making Prs. With a single click it will launch a workspace and automatically:
+You can use Gitpod(an online IDE which is free for Open Source) for working on issues or making PRs. With a single click it will launch a workspace and automatically:
```
Comment (danny007in) [inline .github/CONTRIBUTING.md:31]: rename `PRs` to `PRs (Pull Requests)`

**Review by danny007in — CHANGES_REQUESTED** [review]

**Review by danny007in — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (gurvirlochab) [reply .github/CONTRIBUTING.md]: Resolved that. Thank you

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (gurvirlochab) [reply .github/CONTRIBUTING.md]: Done

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (gurvirlochab) [reply .github/CONTRIBUTING.md]: Renamed that. Thank you

**Pull request merged**